### PR TITLE
updated node.js version

### PR DESCRIPTION
**Pull request by hrajotia**
- added Node.js version `4`, `5` and `6`.
- logging `git`, `mongodb` and `adapt` versions.
- removed caching for `node_modules` directory.
- removed `on_start` from `notifications` as its showing deprecated on [Travis WebLint](https://lint.travis-ci.org/).

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -128,7 +128,7 @@
   },
   "main": "index",
   "engines": {
-    "node": ">=0.10"
+    "node": ">=4.0.0"
```
Comment (brian-learningpool) [inline package.json]: Should this be >= or `4.x`?

Comment (hrajotia) [inline package.json]: Travis seems happy with [Node.js v6.1](https://travis-ci.org/adaptlearning/adapt_authoring/jobs/130845301#L280). So we can also make `"node": ">=4.0.0 <6.x"`.

What do you think?

**Comment by taylortom** [comment]
I'd be tempted to stick with specifying the LTS/@brian-learningpool's suggestion just to be safe.

**Comment by hrajotia** [comment]
@brian-learningpool, @taylortom done!

**Comment by taylortom** [comment]
This all seems ok to me, @brian-learningpool?

File: plugins/content/config/model.schema [hunk 2]
```diff
@@ -125,7 +125,7 @@
           "type": "string",
           "required": true,
           "default": "easeOutQuart",
-          "title": "Hide Easing",
```
Comment (taylortom) [inline plugins/content/config/model.schema]: Should these be the other way round? We have easeOutQuart for the '_showEasing' animation.

Comment (hrajotia) [inline plugins/content/config/model.schema]: yes we have `easeOutQuart` for `_showEasing` and `easeInQuart` for `_hideEasing` [see config.json](https://github.com/adaptlearning/adapt_framework/blob/master/src/course/config.json#L11-L12). but mistakenly both titles was labeled as `Hide Easing` that's why I have corrected it!

**Comment by brian-learningpool** [comment]
+1

**Pull request merged**